pkg/keycloak/realm/phaseHandler.go

```diff
@@ -1,17 +1,17 @@
 package realm
 
 import (
-	"github.com/operator-framework/operator-sdk/pkg/util/k8sutil"
 	"reflect"
 	"strings"
 
+	"github.com/operator-framework/operator-sdk/pkg/util/k8sutil"
+
 	"github.com/integr8ly/keycloak-operator/pkg/apis/aerogear/v1alpha1"
 	"github.com/integr8ly/keycloak-operator/pkg/keycloak"
 	"github.com/integr8ly/keycloak-operator/pkg/util"
 	"github.com/pkg/errors"
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Out of two duplicate imports I personally prefer `metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"`
Also, it is used in code, and build failed because of removal.

Reply:
sorry. it was my editor that changed the imports. Updated not. I will squash the commits